src/SimpleThings/EntityAudit/AuditReader.php

```diff
@@ -819,4 +819,44 @@ public function getEntityHistory($className, $id)
 
         return $result;
     }
+    
+    public function matchRevisions($className, $id, RevisionCriteria $criteria)
+    {
+        $revisions = $this->findRevisions($className, $id);
+        return array_filter($revisions, function($revision) use ($criteria) {
+            return $this->matchCriteria($revision, $criteria);
```

Comment:
`$this` isn't working with PHP 5.3 in this context. But PHP 5.3 support should be dropped anyway (IHMO)